### ISSUE TITLE
How to call R functions directly and supply them JS arrays (as variables)?

### ISSUE THREAD

**Issue by emmansh** [FabrizioSandri/r-integration#6]
Thank you for this package! I have a question about calling R functions as-is from node.js.

I've seen in the readme file the example with R's `max()` function:
```javascript
// In NodeJS
const R = require('r-integration');

let result = R.executeRCommand("max(1,2,3)");
console.log(result);

> [ '3' ]
```

However, I wonder how I would pass an array from node to `max()`. For example:
```javascript
// In NodeJS
const R = require('r-integration');

let myArray = [1, 2, 3, 4, 5]
```
And now I want to take the maximal value of `myArray` using R's built-in `max()`. How should I pass `myArray` to `R.executeRCommand`?

*A possible solution I prefer to avoid*
The way I see it, I *could* do:
```r
# ./scripts/max_script.R

max(input_x)
```
then
```javascript
let result = R.callMethod(" ./scripts/max_script.R", "max", myArray)
```
But this solution demonstrates an unnecessary detour because we already have `max()` set up, and I just want to use it as-is!

By the same token, although question has to with a base R function (e.g., `max()`), the point extends to any function that I want to use as-is. For example, consider[ `fmode()`](https://sebkrantz.github.io/collapse/reference/fmode.html) from `{collapse}` package. In R, I could do:
``` r
library(collapse, warn.conflicts = FALSE)
#> collapse 1.7.2, see ?`collapse-package` or ?`collapse-documentation`

my_vec <- c(1, 1, 3, 4, 5, 5, -6, -6, 2, 2)
fmode(my_vec, ties = "last") # takes the last mode in case there are multiple modes
#> [1] 2
```
So if I'm in node, I'd like to use `collapse::fmode()` as-is, without setting up a special R script:
```javascript
// In NodeJS
const R = require('r-integration');

let myArrayWithDupes = [1, 1, 3, 4, 5, 5, -6, -6, 2, 2]

// something in the spirit of this direct call to fmode:
let result = R.executeRCommand("collapse::fmode", {x: myArrayWithDupes, ties: "last"});
```

Thanks again, wonderful package!

**Comment by FabrizioSandri** [comment]
Good catch! I've just released a new version 1.8.0 and updated the documentation. 

Now you can call a standard function without setting up an external R script, by using the `callStandardMethod` function. 
Update to this new version by executing ` npm install r-integration`.

You can find more about this new function [here](https://github.com/FabrizioSandri/r-integration#call-a-standard-r-function-with-parameters).
Thanks!

**Comment by emmansh** [comment]
Thanks! Your solution already triggered a new issue (see #7 )
:)
